datajunction-server/datajunction_server/api/namespaces.py

```diff
@@ -21,7 +21,7 @@
 
 
 @router.post("/namespaces/{namespace}/", status_code=201)
-def create_a_node_namespace(
```

Comment:
nit: I would keep this `create_a_node_namespace` because this will get automatically translated in the API documentation to capital case and spaces. So "Create a Node Namespace" looks better than "Create Node Namespace".

Reply:
omg, I now understand why this strange naming convention. So... `_a_` infix would only be needed on functions that are exposed in router? I am not sure it is worth the trouble. But if you both think it is then ¯\_(ツ)_/¯

Reply:
@agorajek and @samredai Turns out there's actually another way of doing this with FastAPI:
```
@router.post("/namespaces/{namespace}/", status_code=201, name="Create a Node Namespace")
```
Using `name` lets us customize what's displayed in the Swagger docs

Reply:
I do prefer this way, cause it is more customizable and has less awkward function names.

Reply:
If you are ok, I'll merge this PR and address this separately.